events/0116_gc_intro-coding.html

```diff
@@ -0,0 +1,165 @@
+                <!-- <p class="box"><strong>Girl Develop It</strong> is here to provide affordable programs for adult women interested in learning web and software development in a judgment-free environment.</p> -->
+
+                <!-- Uncomment for Workshops -->
+                <div class="copy--small">
+                    <h4 class="green">Some "rules"</h4>
+                    <ul>
+                        <li>We are here for you!</li>
+                        <li>Every question is important.</li>
+                        <li>Help each other.</li>
+                        <li><strong>Have fun!</strong></li>
+                    </ul>
+                </div>
+
+                <p class="box copy--small"><small><strong>Girl Develop It is dedicated to providing a harrasment free learning experience for <em>everyone</em>.</strong><br />For more information, see our <a href="http://www.girldevelopit.com/code-of-conduct" target="_blank">Code of Conduct</a>.</small></p>
+            </section>
+
+            <!-- Upcoming Events -->
+            <section>
+                <h3>Upcoming Events</h3>
+                <div class="box">
+                    <h3 class="event__title box--small green"><small><span class="blue">Coffee &amp; Code</span></small></h3>
+                    <h3 class="event__title"><small>Saturday, 1/16 @ Amer's Deli</small></h3>
+                    <p><small>10:00am - 1pm &nbsp; | &nbsp; 312 S. State Street</small></p>
+                    <aside class="notes"></aside>
+                </div>
+                <div class="box">
+                    <h3 class="event__title box--small green"><small><span class="blue">New Year, New Headshot!</span></small></h3>
+                    <h3 class="event__title"><small>Wednesday, 1/20 @ Duo Security, Inc.</small></h3>
+                    <p><small>6:00pm - 8pm &nbsp; | &nbsp; 123 N Ashley St, Suite 200</small></p>
+                    <aside class="notes"></aside>
+                </div>
+<!--                 <div class="box">
+                    <h3 class="event__title box--small green"><small><span class="blue">A Day In The Life Series:</span><br />Execs, Owners, Entrepreneiurs, &amp; Do'ers</small></h3>
+                    <h3 class="event__title"><small>Monday, 1/25 @ Nutshell</small></h3>
+                    <p><small>6:30pm - 8pm &nbsp; | &nbsp; 212 S. 5th Ave</small></p>
+                    <aside class="notes"></aside>
+                </div>
+                 <div class="box">
+                    <h3 class="event__title box--small green"><small><span class="blue">Intro to Ruby Workshop</span><br />4-Session Workshop</small></h3>
+                    <h3 class="event__title"><small>On Tuesdays starting 2/2 @ Nutshell</small></h3>
+                    <p><small>6:00pm - 9pm &nbsp; | &nbsp; 212 S. 5th Ave</small></p>
+                   <aside class="notes"></aside>
+                </div>
+   -->         </section>
```

Comment:
They didn't fit on the single page. I'll update to have a second page of GDI events.